### DISCUSSION TITLE
Configuration reload hangs i3

### DISCUSSION THREAD

**Discussion by domsch1988** [i3/i3#6141] [Q&A] [answered]
I have an issue where my configuration won't properly reload. It starts fine and works correctly in any other way, but reloading the config indefinitly hangs i3. I can keep working on the last focused window but not change focus or workspaces or such. Only force killing my session from another TTY allows me to log back in.
Running i3-msg reload from a terminal doesn't produce any output and hangs indefinitly. Killing that command doesn't restore i3 to a working state.

Not even sure how to approach this issue. I would really appreciate some help in troubleshooting this. The current configuration can be found here: https://gitlab.com/domsch1988/domfiles/-/blob/main/.config/i3/config

**Comment by domsch1988** [comment, marked as answer]
As always, find the solution minutes after making an issue for it. So here goes:
Through the official "How to Debbug" Documentation i enabled logging and went through that. Well, turns out i was trying to match a window_type to move a splashscreen to a specific workspace, which isn't recognized by i3. In this case "_NET_WM_WINDOW_TYPE_SPLASH". Why this would completely hang reoading the config, but not fail on starting i3 is beyond me.

Have deleted that line and everything is back to working again.

**Comment by orestisfl** [comment]
Thank you for noticing: https://github.com/i3/i3/pull/6142
